src/screens/PaymentDetailsScreen.tsx

```
@@ -1,29 +1,155 @@
+                  max={1000000}
+                  value={paymentShares.get(member.name) || ''}
+                  onChange={(e) => handleInputChange(member.name, e.target.value)}
+                />
+              </p>
+            </div>
+          </div>
+        </div>
+      ))}
+      <div className="field is-horizontal mt-4 has-background-primary-light p-3">
+        <div className="field-label is-normal is-expanded">
+          <label className="label is-size-5 has-text-weight-bold">Total</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)